### ISSUE TITLE
Change currency and timezone

### ISSUE THREAD

**Issue by nickzasa** [Attendize/Attendize#455]
When changing currency or timezone attendise says

**Comment by nickzasa** [comment]
When setting the currency to south african rand it says account settings updated but it does not change the currency to south african rands

**Comment by nickzasa** [comment]
when we change to mastercard for the payment method and click save it does nothing.

**Comment by jeremyquinton** [comment]
@nickzasa are you using the latest version of Attendize e.g the master branch?

**Comment by jeremyquinton** [comment]
@nickzasa I tested this and the account settings are saving correctly might be something wrong with your setup?

**Comment by getinfo-developer** [comment]
I am also having the same issue. I created ticket with GBP as default currency but later I changed to USD and save it but it is still showing GBP

**Comment by jeremyquinton** [comment]
@nickzasa in the release of Attendize 1.1 and Laravel 5.6 we removed Coinbase and Mastercard as default payment gateways however didn't remove them out the database as configurable options so it seems to have created some confusion. I have removed them 
here. 
https://github.com/Attendize/Attendize/commit/7f7bde0ff58faee26fbeb7915609bc67e38c9f80

**Comment by jeremyquinton** [comment]
@getinfo-developer what you are experiencing is a different issue I'll take a look at this.

**Comment by samdb** [comment]
I’ve been able to replicate inability to change currency. Any events created after changing the currency are fine but it does not retroactively change events that have already been set up in in the previous currency.

**Comment by jeremyquinton** [comment]
@samdb yep thats correct. The issue is that when changing currency from account settings it changes  currency_id on the account table however the events table also has a currency_id that is not updated. 

All the application views(html) reference the currency_id on the events table and not the accounts table.  

When an event is created it is populated from the default currency set in https://github.com/Attendize/Attendize/blob/master/config/attendize.php#L63. 

The question now is if an organiser has multiple events do they need the ability to have the events in different currencies or do we just default to one currency for all events. 

A simple fix would be change the application views to reference the currency from accounts but that then means one currency for all events.

**Comment by samdb** [comment]
@jeremyquinton We may need to look at making both currency and tax per-event options. I have had another request just today for per-event taxes from people that hold events in multiple countries.

I can see this being something I would require myself so understand the need.

**Comment by jimrome** [comment]
What are the currency codes? None of this is documented!
